deploy/src/main/deploy/helm/templates/dispatch-router/config/qrouterd-with-servicebus.yaml

```diff
@@ -0,0 +1,145 @@
+{{- define "hono.qdrouterd-with-servicebus.json" -}}
+[
+  ["router", {
+    "id": "Hono.Example.Router",
+    "mode": "standalone",
+    "workerThreads": 3
+  }],
+
+  ["authServicePlugin", {
+    "name": "Hono Auth",
+    "host": "{{ .Release.Name }}-service-auth",
+    "port": 5671,
+    "sslProfile": "internal"
+  }],
+
+  ["sslProfile", {
+    "name": "external",
+    "protocols": "TLSv1.2",
+    "caCertFile": "/etc/hono/trusted-certs.pem",
+    "certFile": "/etc/hono/qdrouter-cert.pem",
+    "privateKeyFile": "/etc/hono/qdrouter-key.pem"
+  }],
+
+  ["sslProfile", {
+    "name": "internal",
+    "protocols": "TLSv1.2",
+    "caCertFile": "/etc/hono/trusted-certs.pem",
+    "certFile": "/etc/hono/qdrouter-cert.pem",
+    "privateKeyFile": "/etc/hono/qdrouter-key.pem",
+    "uidFormat": "ou"
+  }],
+
+  ["listener", {
+    "sslProfile": "external",
+    "requireSsl": true,
+    "host": "0.0.0.0",
+    "port": 5671,
+    "maxSessionFrames": 200,
+    "linkCapacity": 100,
+    "authenticatePeer": true,
+    "saslMechanisms": "PLAIN",
+    "saslPlugin": "Hono Auth"
+  }],
+
+  ["listener", {
+    "host": "0.0.0.0",
+    "port": 5672,
+    "maxSessionFrames": 200,
+    "linkCapacity": 100,
+    "authenticatePeer": true,
+    "saslMechanisms": "PLAIN",
+    "saslPlugin": "Hono Auth"
+  }],
+
+  ["listener", {
+    "sslProfile": "internal",
+    "requireSsl": true,
+    "host": "0.0.0.0",
+    "port": 5673,
+    "maxSessionFrames": 500,
+    "linkCapacity": 100,
+    "authenticatePeer": true,
+    "saslMechanisms": "EXTERNAL"
+  }],
+
+  ["connector", {
+    "sslProfile": "external",
```

Comment:
I am not sure if this will work out-of-the-box with ServiceBus using a certificate that has been signed by a public CA, because the *external* SSL profile is configured to use *trusted-certs.pem* as the trust store, which includes only the CA certs used by the Hono example. Have you actually tried this?

Reply:
I does work, for sure. Let me see if I can figure out why.

Reply:
I could not figure it out exactly. I guess there are two options, either the router is not trying to TLS client authenticate as SASL is configured as well or SB is plain ignoring it just taking the SASL plain auth.

Anyhow, I created a new profile only defining the TLS version which is sufficient. Makes it clearer that way.